src/app/pages/home/home.page.html

```diff
@@ -1,20 +1,64 @@
-<ion-header [translucent]="true">
-  <ion-toolbar>
+<ion-header no-border>
+  <ion-toolbar color=success>
     <ion-title>
-      Blank
+      Reportes
     </ion-title>
   </ion-toolbar>
 </ion-header>
 
-<ion-content [fullscreen]="true">
-  <ion-header collapse="condense">
-    <ion-toolbar>
-      <ion-title size="large">Blank</ion-title>
-    </ion-toolbar>
-  </ion-header>
+<ion-content padding>
 
-  <div id="container">
-    <strong>Ready to create an app?</strong>
-    <p>Start with Ionic <a target="_blank" rel="noopener noreferrer" href="https://ionicframework.com/docs/components">UI Components</a></p>
+  <div class="form-container ion-padding">
+    <ion-list>
+      
+      <ion-item>
+        <ion-label position="floating">Escribe tu reporte aquí:</ion-label>
+        <ion-textarea></ion-textarea>
+      </ion-item>
+      <ion-item>
+        <ion-label>Aula a reportar: </ion-label>
+        <ion-select  ok-text="Seleccionar" cancel-text="Cancelar">
+          <ion-select-option value="au1">Aula 1</ion-select-option>
+          <ion-select-option value="au2">Aula 2</ion-select-option>
+          <ion-select-option value="au3">Aula 3</ion-select-option>
+          <ion-select-option value="au4">Aula 4</ion-select-option>
+          <ion-select-option value="audi1">Auditorio 1</ion-select-option>
+          <ion-select-option value="audi2">Auditorio 2</ion-select-option>
+          <ion-select-option value="audi3">Auditorio 3</ion-select-option>
+          <ion-select-option value="audi4">Auditorio 4</ion-select-option>
+          <ion-select-option value="LabRed">Lab. Redes</ion-select-option>
+        </ion-select>
+      </ion-item>
+  
+  
+    </ion-list>
+  
+    <ion-row>
+      <ion-col size="6">
+        <ion-button expand="full"
+                    shape="round"
+                    color=warning>
+          <ion-icon slot="start" name="camera"></ion-icon>
+          Cámara
+        </ion-button>
+      </ion-col>
+      <ion-col size="6">
+        <ion-button expand="full"
+                    shape="round"
+                    color=warning>
+          <ion-icon slot="start" name="images"></ion-icon>
+          Galería
+        </ion-button>
+      </ion-col>
+    </ion-row>
+  
+    <ion-row>
+      <ion-col size="3" *ngFor="let image of tempImages">
```

Comment:
If you haven't created the `tempImages` variable yet in the file _home.page.ts_, don't use the ngFor directive at the moment.

Reply:
Sure